### PR TITLE
Python 3: port webappapis and cross-origin tests in html

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

File: html/cross-origin-opener-policy/reporting/resources/report.py [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 def main(request, response):
-    response.headers.set('Cache-Control', 'no-cache, no-store, must-revalidate');
+    response.headers.set(b'Cache-Control', b'no-cache, no-store, must-revalidate')
 
-    key = 0;
-    if 'endpoint' in request.GET:
-        key = uuid.uuid5(uuid.NAMESPACE_OID, request.GET['endpoint']).get_urn()
+    key = 0
+    if b'endpoint' in request.GET:
+        key = uuid.uuid5(uuid.NAMESPACE_OID, request.GET[b'endpoint']).get_urn()
```
Comment (jdm) [inline html/cross-origin-opener-policy/reporting/resources/report.py:8]: get_urn() doesn't exist in python3; however, both python2 and python3 have a `urn` property: https://docs.python.org/3/library/uuid.html#uuid.UUID.urn https://docs.python.org/2/library/uuid.html#uuid.UUID.urn

**Review by jdm — DISMISSED** [review]

File: html/cross-origin-opener-policy/reporting/resources/report.py (hunk 1 above)
Comment (ziransun) [reply html/cross-origin-opener-policy/reporting/resources/report.py]: Updated code. Thank you!

**Comment by jdm** [comment]
This might need to be rebased? I'm not sure what to make of the taskcluster failure.

**Comment by ziransun** [comment]
> This might need to be rebased? I'm not sure what to make of the taskcluster failure.

Code updated. Thank you!

**Review by jdm — APPROVED** [review]

**Pull request merged**